test/src/test_cases/edx-schools-frontend/test-school-inbox.js

```diff
@@ -11,10 +11,13 @@ import { createTestExchange } from "../../helpers/secure-exchange-utils";
 
 import log from "npmlog";
 import Inbox from "../../page_models/inbox";
+import HamburgerMenuPage from "../../page_models/common/hamburgerMenuPage";
+import hamburgerMenuPage from "../../page_models/common/hamburgerMenuPage";
```

Comment:
Little confused by this one, I don't think we need two hamburger menu imports?

Reply:
I'm not sure why it's there twice... I'll remove the second one.